New version: OrthogonalSphericalShellGrids v0.1.4

JuliaRegistrator opened the pull request:
- Registering package: OrthogonalSphericalShellGrids
- Repository: https://github.com/CliMA/OrthogonalSphericalShellGrids.jl
- Created by: @glwagner
- Version: v0.1.4
- Commit: 379d0de985a0927c026e6c3e43b2ebd163e054aa
- Reviewed by: @glwagner
- Reference: https://github.com/CliMA/OrthogonalSphericalShellGrids.jl/commit/379d0de985a0927c026e6c3e43b2ebd163e054aa#commitcomment-147072593
- Description: 🌐 Recipes and tools for generating spherical shell grids for ocean simulations with Oceananigans
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd644ea911821cd8c6500ba68de62f2b6aa3c1d3a9ae0ed2286e54d99c2d779daa433a1db7fb52a272210abcecc57a196bf5e593f02613d2f5baba3bf740336795abb2c02ed8c655286c98b455c15ae7eefad4784f99817567c325d3a08ffb08ca32278e5bbcefc69599a5e086230b328c3f8de727acdac0e670fe0ad35d5cae2dbcd400fcb7f91a054cb0f6dd03dd5d50383fa5db7ac4c72a047dcd8ab7066254f5fbf17cfc51f3b0f99215055a143a82db87f03f98d9b01dc605a10b30372bbd8 -->

navidcy commented:
can we retrigger the automerge?

glwagner commented:
@navidcy looks like the automerge guidelines aren't met

navidcy commented:
Yes. I’m confused

glwagner commented:
this clarifies https://juliaregistries.github.io/RegistryCI.jl/stable/guidelines/#Upper-bounded-%5Bcompat%5D-entries

navidcy commented:
@glwagner, if you edit your comments to add `[noblock]` then the PR will merge I believe! [noblock]

JuliaRegistrator committed:
New version: OrthogonalSphericalShellGrids v0.1.4

UUID: c2be9673-fb75-4747-82dc-aa2bb9f4aed0
Repo: https://github.com/CliMA/OrthogonalSphericalShellGrids.jl.git
Tree: 9ef11c020889e7aa3ae38af517abefe388ff3ecd

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
